src/api/basics.rst

```diff
@@ -141,7 +141,7 @@ Request Headers
 
   The returned headers are:
 
-  .. code-block:: http
+  .. code-block:: none
```

Comment:
The http language is provided by httpdomain extension. Why sphinx 1.3.5+ fails to use it? I think the proper fix need to fix the syntax highlight, not turn it off just because some bug.